### DISCUSSION TITLE
Would it be better to build the `protobuf` package using `cmake`?

### DISCUSSION THREAD

**Discussion by Cryspia** [Homebrew/discussions#3117] [Everyday usage]
Presently, `protobuf` is built using `configure` according to the [ruby installation file](https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/protobuf.rb).
I would suggest to use `cmake` instead, or the `lib/cmake` folder will be missing, thus creating inconvenience for developers who are working on a project importing protobuf and built with cmake.

**Comment by mistydemeo** [comment]
We're following the officially-recommended installation steps from the protobuf documentation, which suggests using `configure`/`autoconf`: https://github.com/protocolbuffers/protobuf/blob/master/src/README.md#c-installation---unix
